### PR TITLE
Allow to install pipeline-service automatically

### PR DESCRIPTION
**Pull request by Michkov**
- rename default workspace to redhat-pipeline-service-compute

### CODE REVIEW

File: hack/preview.sh [hunk 1]
```diff
@@ -33,13 +33,16 @@ if [ -n "$PIPELINE_SERVICE_IDENTITY_HASH" ]; then
   IDENTITY_HASHES="pipeline-service:$PIPELINE_SERVICE_IDENTITY_HASH"
 else
   KUBECONFIG=${KCP_KUBECONFIG} kubectl ws ${ROOT_WORKSPACE}
-  if KUBECONFIG=${KCP_KUBECONFIG} kubectl ws pipeline-service; then
+  if KUBECONFIG=${KCP_KUBECONFIG} kubectl ws redhat-pipeline-service-compute; then
     IDENTITY_HASH=$(oc get apiexport kubernetes --kubeconfig ${KCP_KUBECONFIG} -o jsonpath='{.status.identityHash}')
     IDENTITY_HASHES="pipeline-service:$IDENTITY_HASH"
+  elif [ "$PIPELINE_SERVICE_LOCAL_DEPLOY" == "true" ]; then
+    KUBECONFIG={CLUSTER_KUBECONFIG} ./hack/install-pipeline-service.sh
```
Comment (MatousJobanek) [inline hack/preview.sh:40]: but in this case, we need to wait until the APIExport is created and then take the identity hash right?
something like this:
```suggestion
    KUBECONFIG={CLUSTER_KUBECONFIG} ./hack/install-pipeline-service.sh
    IDENTITY_HASH=$(oc get apiexport kubernetes --kubeconfig ${KCP_KUBECONFIG} -o jsonpath='{.status.identityHash}')
    IDENTITY_HASHES="pipeline-service:$IDENTITY_HASH"
```

Comment (Michkov) [reply hack/preview.sh]: updated, thanks (also fixed install-pipeline-service)

File: hack/install-pipeline-service.sh [hunk 2]
```diff
@@ -4,20 +4,20 @@ ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/..
 
 source ${ROOT}/hack/flags.sh "The install-pipeline-service.sh installs Pipeline Service for development and testing on non-production clusters / kcp instances."
 MODE=preview parse_flags $@
-
+KCP_WORKSPACE=redhat-pipeline-service-compute
```
Comment (MatousJobanek) [inline hack/install-pipeline-service.sh:7]: let's keep the var names consistent 
```suggestion
PIPELINE_SERVICE_WORKSPACE=redhat-pipeline-service-compute
```

Comment (Michkov) [reply hack/install-pipeline-service.sh]: Updated

**Review by MatousJobanek — APPROVED** [review]

**Pull request merged**